docs/readmes/orc8r/upgrade_1_6.md

```diff
@@ -0,0 +1,92 @@
+---
+
+# Upgrade to v1.6
+
+This guide covers upgrading Orchestrator deployments from v1.5 to v1.6. Upgrades which skip versions (e.g. v1.4 to v1.6) are not explicitly documented or supported at this time.
+
+The v1.6 upgrade follows a standard procedure
+
+- Upgrade Terraform modules
+- Terraform install
+
+## Prerequisites
+
+### Terraform version
+
+This document was tested using Terraform version `0.15.5`. You can use [`tfenv`](https://github.com/tfutils/tfenv) to install a particular Terraform version.
+
+### Terraform state
+
+If you are using local Terraform state (the default), ensure all Terraform state files are located in your working directory before proceeding.
+
+To verify this, `terraform show` should list existing state, rather than outputting `No state`.
+
+### Postgres version
+
+[AWS is terminating support for Postgres 9.6 on November 11, 2021](https://aws.amazon.com/blogs/database/upgrading-from-amazon-rds-for-postgresql-version-9-5/), in accordance with the [Postgres versioning policy](https://www.postgresql.org/support/versioning/). Postgres 9.6 was the default Orc8r DB target for the past several releases.
+
+As of Magma release v1.6, the new default Orc8r DB target is Postgres 12.6. *If you are upgrading from a previous release, you will need to handle this change*.
+
+The recommended path is
+1. Pin the Terraform DB engine to your existing version (likely `9.6.15`)
+2. Separately, at a later date, [use Terraform to upgrade to Postgres 12.6](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance)
+
+Step (1) above is captured in the [upgrade](#upgrade) section. Step (2) above is non-actionable for the v1.6 upgrade. [We will provide forthcoming guidance on performing the DB upgrade](https://github.com/magma/magma/issues/7871), which [will require minor DB downtime](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_UpgradeDBInstance.PostgreSQL.html).
+
+### Orc8r artifacts
+
+As of the v1.6 release, Orc8r artifacts are now available on a publicly-hosted artifactory. This means you no longer need to build from source or manage your own artifactory.
+
+See the [host custom artifacts](#host-custom-artifacts) section if you would rather build your own artifacts.
+
+## Upgrade
+
+### 1. Update Terraform modules
+
+Update your root Terraform module
+
+```terraform
+# This will likely be found in main.tf
+
+module orc8r {
+  source = "github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-aws?ref=v1.6"
+  # ...
+  # Your current Postgres engine version will be 9.6.15, unless
+  # you previously overrode it
+  orc8r_db_engine_version = "9.6.15"  # pin Postgres engine version
+}
+
+module orc8r-app {
+  source = "github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-helm-aws?ref=v1.6"
+  # ...
+  # IMPORTANT: delete all docker_ and helm_ lines
+  # ...
+  orc8r_tag = "v1.6.0"
+}
+```
+
+Pull in the module changes
+
+```bash
+terraform init --upgrade
+terraform refresh
+```
```

Comment:
We added new options in terraform including sns notifications for RDS alerts, setting elasticsearch_disk_threshold we can add that either here or in another document customizing terraform.

Reply:
Added notes